recipes/libnettle/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nettle" %}
+{% set version = "3.3" %}
+{% set sha256 = "46942627d5d0ca11720fec18d81fc38f7ef837ea4197c1f630e71ce0d470b11e" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]  
+
+requirements:
+  build:
+    - m4
+    - gcc
+  run:
+    - gmp
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX nettle
+
+about:
+  home: https://www.lysator.liu.se/~nisse/nettle/
+  license: GPL 2 and 3 
+  license_family: GPL
+  summary: 'Nettle is a low-level cryptographic library that is designed to fit easily in more or less any context'
+  description: 'Nettle is a low-level cryptographic library that is designed to fit easily in more or less any context: In crypto toolkits for object-oriented languages (C++, Python, Pike, ...), in applications like LSH or GNUPG, or even in kernel space. Simple, fast, extensible JSON encoder/decoder for Python'
```

Comment:
Could you please wrap this line like is done in the [`example`]( https://github.com/conda-forge/staged-recipes/blob/e438484a22aa68d419a96519032504ea6c66b247/recipes/example/meta.yaml#L59-L63 )?